public/js/src/module/settings.js

```diff
@@ -63,13 +63,13 @@ settings.languageOverrideParameter = queryParams.lang ? {
 settings.maxSize = DEFAULT_MAX_SIZE;
 
 // add type
-if ( window.location.pathname.indexOf( '/preview' ) === 0 ) {
+if ( window.location.pathname.includes( '/preview/' ) ) {
```

Comment:
This wouldn't identify https://enketo.getodk.org/preview?form=https://xlsform.getodk.org/downloads/h6cp6h3m/mini_form.xml (as redirected to in  https://getodk.org/xlsform/) as a preview. That may not actually matter in reality, but let's add `|| window.location.pathname.endsWith('/preview')` just to make sure we don't run into issues (in the future).